src/app/lib/config/services/serbian.js

```diff
@@ -234,6 +239,11 @@ export const service = {
     locale: `sr-cyrl`,
     script: cyrillicAndLatin,
     datetimeLocale: `sr-cyrl`,
+    scriptLink: {
+      variant: 'cyr',
```

Comment:
`scriptLink` is already under the `cyr` variant key in the service config. Can we remove this here.

Reply:
Removed the `variant` attribute.